spec/features/user_edits_profile_spec.rb

```diff
@@ -52,7 +51,7 @@
     visit edit_user_registration_path(user)
 
     fill_in "user[age]", with: "301"
-    click_button "Update"
+    first('#account > div:nth-child(4) > div').click_button "Update"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.